src/Component/Navbar/NavbarStore/NavbarStore.js

```diff
@@ -91,82 +89,136 @@ class NavbarStore extends Component {
         });
       });
     this.props.history.push("/cart");
-  }
+  };
 
   render() {
     console.log(this.state.isUnisexBorder);
-    const { isOotdBorder, isStoreBorder, isUnisexBorder, isHomeBorder, cartList } = this.state;
+    const {
+      isOotdBorder,
+      isStoreBorder,
+      isUnisexBorder,
+      isHomeBorder,
+      cartList,
+    } = this.state;
     return (
       <>
-      <div className="navBarWrapper">
-        <nav className="topNavBar">
-          <div className="leftNavBar">
-            <a className="navLogo" href="/">
-              <img src='/images/navLogo.png'/>
-            </a>
-            <div className="leftMenuWrapper">
-              <ul className="leftMenuList">
-                <li value="feed"
-                    className="leftMenu">
-                  <span  className={isOotdBorder ? "leftMenuOotdAct":"leftMenuOotd"} onClick={this.handleClickOotd}>#OOTD</span>
-                </li>
-                <li value="feed"
-                    className="leftMenu">
-                  <span className={isStoreBorder ? "leftMenuStoreAct":"leftMenuStore"} onClick={this.handleClickStore}>STORE</span>
-                </li>
-              </ul>
+        <div className="navBarWrapper">
+          <nav className="topNavBar">
+            <div className="leftNavBar">
+              <a className="navLogo" href="/">
+                <img src="/images/navLogo.png" />
+              </a>
+              <div className="leftMenuWrapper">
+                <ul className="leftMenuList">
+                  <li value="feed" className="leftMenu">
+                    <span
+                      className={
+                        isOotdBorder ? "leftMenuOotdAct" : "leftMenuOotd"
```

Comment:
```suggestion
                        isOotdBorder ? "leftMenuOotd act" : "leftMenuOotd"
```
이렇게 복수의 클래스명을 적용할 수 있기 때문에 공통 스타일을 재활용할 수 있을 것 같습니다